portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,142 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <meta charset="UTF-8" />
+    <link rel="shortcut icon" href="assets/cartoon-edited.png" />
+    <link rel="stylesheet" type="text/css" href="styles/styles.css" />
+    <link
```

Comment:
nit: typically we put the first element attribute so that it is on the same line as the starting tag.

Reply:
Applies elsewhere

Reply:
Got it.